requirements.txt

```diff
@@ -2,3 +2,4 @@ Twisted>=13.2.0
 git+git://github.com/graphite-project/whisper.git#egg=whisper
 txAMQP
 cachetools
+urllib3
```

Comment:
is there a rationale for urllib3 vs. request ? (curious)

Reply:
Because we already use it in graphite-web

Reply:
fwiw, requests is a wrapper around urllib3

Reply:
agreed, `requests` is just extra dependency here.